schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a web page context",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "gdpr",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+
+    "type": "object",
+    "properties": {
+        "basisForProcessing": {
+          "enum": ["consent", "contract", "legal_obligation", "vital_interest", "public_task", "legitimate_interests"],
```

Comment:
Should be `vital_interests` note the trailing s for consistency with legitimate_interests. Confirmed on https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/

Reply:
Thanks for flagging @alexanderdean , fixed now. :)